packages/flutter_driver/lib/src/driver/frame_request_pending_latency_summarizer.dart

```diff
@@ -0,0 +1,65 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'percentile_utils.dart';
+import 'timeline.dart';
+
+/// Event name for frame request pending timeline events.
+const String kFrameRequestPendingEvent = 'Frame Request Pending';
+
+/// Summarizes [TimelineEvents]s corresponding to [kFrameRequestPendingEvent] events.
+///
+/// `FrameRequestPendingLatency` is the time between when a new frame is requested
+/// to when the frame starts getting built by the Flutter Engine.
```

Comment:
I might be a little more specific on what "starts getting built" means, in terms of specific engine functionality.

Reply:
Hm, should I refer to specific functions in the engine?

Reply:
Yeah, might as wel

Reply:
practically speaking, if you're not familiar with the engine code it will be difficult to interpret this number.

Reply:
Updated to refer to Animator::RequestFrame and Animator::BeginFrame